### ISSUE TITLE
Make mastercontainer and nextcloud container more secure by setting listen.allowed_clients

### ISSUE THREAD

**Issue by szaimen** [nextcloud/all-in-one#3172]
- [x] mastercontainer: https://github.com/nextcloud/all-in-one/pull/3317

```
/usr/local/etc/php-fpm.d/www.conf

;listen.allowed_clients = 127.0.0.1,::1
```

- [ ] nextcloud container: https://github.com/nextcloud/all-in-one/pull/3318
```
/usr/local/etc/php-fpm.d/www.conf

;listen.allowed_clients = ipv4-of-apache;ipv6-of-apache,ipv4-of-mastercontainer-ipv6-of-mastercontainer,localhost,notify-push,dsp
```

**Comment by Zoey2936** [comment]
notify push connects to php-fpm?

**Comment by Zoey2936** [comment]
but does it not only do a `nc` check? I think it wouldn't even see if there is no permission

**Comment by szaimen** [comment]
> but does it not only do a `nc` check? I think it wouldn't even see if there is no permission

It does, I tested and it shows even when only doing a nc check

**Comment by szaimen** [comment]
> for the mastercontainer: what about switching to unix sockets?

This would be an alternative idea 👍

**Comment by szaimen** [comment]
a PR would be welcome, btw! :)

**Comment by szaimen** [comment]
This is now released with v7.4.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
